src/client/monitor/Monitor.tsx

```diff
@@ -65,12 +70,14 @@ export function Monitor({fullScreen, requestFullScreen}: MonitorProps): ReactEle
   const traysAdded = !isEmpty(trays)
   const success = isEmpty(projects)
 
-  const monitorClassNames = cn(styles.monitor, {
-    [styles.fullscreen]: fullScreen
-  })
+  document.onmousemove = () => {
+      setMonitorClassNames({
+        ...monitorClassNames, [styles.inactive]: false
```

Comment:
Nothing is ever setting the inactive style back to `true`, so once the mouse is moved the mouse cursor will never be hidden again.